cypress/page/address-step.page.ts

```diff
@@ -1,12 +1,14 @@
 class AddressStepPage {
   private processAddress: string;
+  private address: string;
 
   constructor() {
-    this.processAddress = "processAddress";
+    this.address = "Proceed to checkout";
```

Comment:
is this used in any place? if not, you can remove it

Reply:
Sorry, I forgot it :P